wp-content/themes/cambodia-ict-camp/inc/template-parts/content-list-posts.php

```diff
@@ -6,66 +6,69 @@
 $no_blog_image = '';
 ?>
 
-<article id="post-<?php the_ID(); ?>" <?php post_class(); ?>>
-    <div class="content-wrapper">
-        <?php
-        // $thumbnail = $event_star_customizer_all_values['event-star-blog-archive-img-size'];
-        $thumbnail = 'full';
-
-        if( has_post_thumbnail() && 'disable' != $thumbnail ):
-            ?>
-            <!--post thumbnal options-->
-            <div class="image-wrap">
-                <div class="post-thumb">
-                    <a href="<?php the_permalink(); ?>">
-                        <?php the_post_thumbnail( $thumbnail ); ?>
-                    </a>
-                </div><!-- .post-thumb-->
-            </div>
+<div class="col-sm-6 col-md-4">
+    <article id="post-<?php the_ID(); ?>" <?php post_class('no-border-bottom'); ?>>
+        <div class="content-wrapper">
             <?php
-        else:
-            $no_blog_image = 'no-image';
-        endif;
-        ?>
+            // $thumbnail = $event_star_customizer_all_values['event-star-blog-archive-img-size'];
+            $thumbnail_size = 'full';
```

Comment:
I won't suggest to set full value by default.

Reply:
I set this value to `full` because I used the CSS to display the image thumbnail with `background-image`.

Reply:
Are those images required in full size? The post/image will be showed in 3 grid, So it won't require to show image is full size which will cause the page loads slowly.

Reply:
The file is not used anymore as it is replaced with `content-grid-3-cols.php`